WordPress/src/main/java/org/wordpress/android/ui/jetpack/scan/ScanListItemState.kt

```diff
@@ -13,9 +13,14 @@ sealed class ScanListItemState(override val type: ViewType) : JetpackListItemSta
         ViewType.THREATS_HEADER
     )
 
+    data class ThreatDateItemState(val text:UiString):ScanListItemState(
+        ViewType.DETECTED_DATE
+    )
```

Comment:
Suggestion: 1. Add spaces around colon
2. Fit into one line

```suggestion
    data class ThreatDateItemState(val text: UiString) : ScanListItemState(ViewType.DETECTED_DATE)
```

Reply:
I've format the code as requested.

Reply:
`ThreatDateItemState(val text:UiString)` 

Space after colon is still missing 😅. We can check this by using `ktlint` locally, let me know if you need any help with this :).

Reply:
Thanks for enlightening me 👍, I am not familiar with `ktlint` 😅. When I check with `ktlint` , there are other warnings in scan package, should I fix them all or fix only the one you mention? 

Example warnings:
`
 68:1 Unexpected indentation (12) (should be 8)
  69:1 Unexpected indentation (12) (should be 8)
  70:1 Unexpected indentation (12) (should be 8)
  71:1 Unexpected indentation (12) (should be 8)
  72:1 Unexpected indentation (16) (should be 12)
  73:1 Unexpected indentation (20) (should be 16)
  74:1 Unexpected indentation (16) (should be 12)
  75:1 Unexpected indentation (20) (should be 16)
  76:1 Unexpected indentation (16) (should be 12)
  77:1 Unexpected indentation (12) (should be 8)
  80:1 Unexpected indentation (12) (should be 8)
  83:1 Unexpected indentation (12) (should be 8)
  84:1 Unexpected indentation (16) (should be 12)
  85:1 Unexpected indentation (12) (should be 8)
  88:1 Unexpected indentation (12) (should be 8)
  89:1 Unexpected indentation (20) (should be 12)
  90:1 Unexpected indentation (24) (should be 16)
  91:1 Unexpected indentation (24) (should be 16)
  92:1 Unexpected indentation (24) (should be 16)
  93:1 Unexpected indentation (24) (should be 16)
  94:1 Unexpected indentation (20) (should be 12)
  104:1 Unexpected indentation (12) (should be 8)
  105:1 Unexpected indentation (16) (should be 12)
  106:1 Unexpected indentation (16) (should be 12)
  107:1 Unexpected indentation (16) (should be 12)
  108:1 Unexpected indentation (12) (should be 8)

`

Reply:
I don't see these indentation warnings if I run `gradlew` `ktlint` command, there must be some configuration issue or these might just be warnings and not errors, we can figure out and discuss on Slack if you want. I'd suggest to fix only the one related to the mentioned line, I don't think CI will allow to merge this PR otherwise.